reasoning_gym/algorithmic/game_of_life.py

```diff
@@ -15,6 +15,7 @@ class GameOfLifeConfig:
     grid_size_x: int = 10
     grid_size_y: int = 10
     filled_cells: int = 100  # actually a max
+    debug_fill: bool = False  # Debugging/test fill pattern, you can ignore this
```

Comment:
What is the idea behind `debug_fill`? Does it have a function or could it be removed?